mkosi/__init__.py

```diff
@@ -4054,12 +4054,17 @@ def install_unified_kernel(
                 image_id = f"mkosi-{args.distribution}"
                 partlabel = None
 
+            # See https://systemd.io/AUTOMATIC_BOOT_ASSESSMENT/
+            boot_count: str = ""
```

Comment:
```suggestion
            boot_count = ""
```
Currently travelling, so I can't check this right away, but mypy should figure this type out by itself. Explicitly adding types is usually only necessary, when annotating empty containers.

Reply:
Done